### ISSUE TITLE
Angular CLI build optimizer breaks core modules

### ISSUE THREAD

**Issue by kwilliams-curago** [ionic-team/ionic-native#2657]
**I'm submitting a ...**  (check one with "x")
[ X ] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://ionicworldwide.herokuapp.com/

**Current behavior:**

The Angular CLI Build Optimizer does not work with ionic-native/core@5.0.0-beta.14.

@angular/cli@6.0.7
@angular/core@6.0.7
@ionic-native/core@5.0.0-beta.14

```
ERROR in ./node_modules/@ionic-native/core/index.js
Module not found: Error: Can't resolve './decorators' in ',/node_modules/@ionic-native/core'
ERROR in ./node_modules/@ionic-native/core/index.js
Module not found: Error: Can't resolve './plugin' in './node_modules/@ionic-native/core'
ERROR in chunk main [initial]
[name].[chunkhash:20].js
./node_modules/@angular-devkit/build-optimizer/src/build-optimizer/webpack-loader.js??ref--17-0!./node_modules/@ngtools/webpack/src/index.js!/./src/main.ts f0e10085353b0e3cca777354c66a3189
Cannot read property 'isProvided' of null
```

**Expected behavior:**

Webpack should be able to resolve the modules properly

**Comment by danielsogl** [comment]
Should be fixed.

**Comment by kwilliams-curago** [comment]
This does not seem to be fixed with 5.0.0-beta.17 

I am still receiving the same result with the build optimizer enabled.
